pkg/utils/kubernetes/crd.go

```diff
@@ -111,3 +113,13 @@ func DecodeCRD(crdYAML string) (*apiextensionsv1.CustomResourceDefinition, error
 	}
 	return crd, nil
 }
+
+// EncodeCRD encodes a CRD into YAML.
+func EncodeCRD(crd *apiextensionsv1.CustomResourceDefinition) (string, error) {
```

Comment:
Why not use https://github.com/gardener/gardener/blob/778012a7925aea86e8ba4a31d51e1428e550bd9e/pkg/utils/kubernetes/serialize.go#L20?